charts/opensearch-operator/templates/UidSecret.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.UpgradeChecker.env }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: opensearch-operator-uid
+type: Opaque
+data:
+  secretKey: {{ randAlphaNum 15 | b64enc | quote }}
```

Comment:
It is generally not recommended to use random values in Helm charts, since deployment tools such as Flux or ArgoCD will see this as changed in each reconciliation loop, and will therefore try to upgrade the chart all the time.
Instead, it could be possible to use some sort of hash of some configuration in the chart.

Reply:
Hey, how would you recommend to implement that logic ?